scripts/hooks/pre-commit

```diff
@@ -21,11 +21,11 @@ function exit_with_errors {
 
 cd aion
 
-printf "Running dialyzer, may take a while if PLT compilation is needed...\n"
-mix dialyzer --halt-exit-status >"$TMP" 2>/dev/null || {
-  print_error "[PRE-COMMIT] Dialyzer has failed with the following result:"
-  exit_with_errors
-}
+#printf "Running dialyzer, may take a while if PLT compilation is needed...\n"
+#mix dialyzer --halt-exit-status >"$TMP" 2>/dev/null || {
+#  print_error "[PRE-COMMIT] Dialyzer has failed with the following result:"
+#  exit_with_errors
+#}
```

Comment:
Why are we disabling dialyzer?

Reply:
I thought about disabling it, because right now we backend is changing very dynamically and basically every PR means editing the `ignore_warnings` file and not really editing the type specs (we don't even have type specs in the main files, so I think it could be better to introduce the dialyzer for good when we are able to add type specs everywhere - especially in the channel part). 

If you want I can remove it, I can also comment it out as a separate PR. This is just a suggestion to remove a tool that's only slowing us down right now and not really bringing any value, because we ignore its output.